TTHAnalysis/python/plotter/ttH-multilepton/mca-includes/mca-2lss-convs.txt

```diff
@@ -6,12 +6,12 @@ Convs : WJetsToLNu_LO : xsec : ((LepGood1_mcMatchId==0 && LepGood1_mcPromptGamma
 #Convs : W2JetsToLNu_LO_skimSameSign : xsec : ((LepGood1_mcMatchId==0 && LepGood1_mcPromptGamma==1) || (LepGood2_mcMatchId==0 && LepGood2_mcPromptGamma==1)) ; FillColor=ROOT.kOrange
 #Convs : W3JetsToLNu_LO_skimSameSign : xsec : ((LepGood1_mcMatchId==0 && LepGood1_mcPromptGamma==1) || (LepGood2_mcMatchId==0 && LepGood2_mcPromptGamma==1)) ; FillColor=ROOT.kOrange
 #Convs : W4JetsToLNu_LO_skimSameSign : xsec : ((LepGood1_mcMatchId==0 && LepGood1_mcPromptGamma==1) || (LepGood2_mcMatchId==0 && LepGood2_mcPromptGamma==1)) ; FillColor=ROOT.kOrange
-Convs : DYJetsToLL_M50 : xsec : ((LepGood1_mcMatchId==0 && LepGood1_mcPromptGamma==1) || (LepGood2_mcMatchId==0 && LepGood2_mcPromptGamma==1)) ; FillColor=ROOT.kOrange
```

Comment:
Please don't commit changes that would affect everyone, just leave them in the working directory.
This needs to be reverted

Reply:
reverted